tests/unit/test_plugins_pgconf.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+
+def test_human_to_number():
+    from temboardagent.plugins.pgconf.functions import human_to_number
+
+    assert 2 == human_to_number('2ms', 'ms')
+    with pytest.raises(Exception):
+        assert 0.2 == human_to_number('0.2ms', 'ms')
```

Comment:
I don't get this. Is the `human_to_number()` call expected to raise (in which case, the assertion should be removed)? or is it that the assertion is expected not to be verified (in which case, using `!=` would be more appropriate)?

Reply:
Good catch! I will open a new PR for this minor fix.